<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_course</class>
 <widget class="QDialog" name="add_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>必修</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选修</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>添加课程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>课程编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>课程学分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
